src/utils.jl

```diff
@@ -1640,4 +1641,106 @@ function LithostaticPressure!(Plithos::Array{T,N}, Density::Array{T,N}, dz::Numb
     Plithos[:] = reverse!(cumsum(reverse!(Plithos),dims=N))
 
     return nothing
+end
+
+"""
+    inPolygon!(PolyX::Vector, PolyY::Vector, X::Matrix, Y::Matrix, INSIDE::Matrix; fast=false)
+
+Checks if points given by matrices `X` and `Y` are in or on (both cases return true) a polygon given by `PolyX` and `PolyY`. Boolean `fast` will trigger faster version that may miss points that are exactly on the edge of the polygon. Speedup is a factor of 3.
+
+"""
+function inPolygon!(PolyX::Vector{T}, PolyY::Vector{T}, X::Matrix{T}, Y::Matrix{T}, INSIDE::Matrix{Bool}; fast=false) where T <: Real
```

Comment:
the naming convention for in-place Julia functions is that the array that is being changed should be listed first.
So this should be renamed as:
```Julia
function inPolygon!(INSIDE::Matrix{Bool}, PolyX::Vector{T}, PolyY::Vector{T}, X::Matrix{T}, Y::Matrix{T}; fast=false) where T <: Real
```

Reply:
I changed it around but now one of the tests failed. It says it is in IO and sounds like a network problem of the test server. Do you understand what is going on there?
![image](https://github.com/JuliaGeodynamics/GeophysicalModelGenerator.jl/assets/84498252/6cee8119-1715-4c7a-8161-58df3fe932ad)

Reply:
yes, we have this frequently. Downloading remote data is sometimes fragile. restarting the CI often solves it.